source/postprocess/dynamic_topography.cc

```diff
@@ -110,9 +117,14 @@ namespace aspect
               // for each of the quadrature points, evaluate the
               // stress and compute the component in direction of the
               // gravity vector
+              
+              double dynamic_topography_x_volume = 0;
+              double volume = 0;
+              Point<dim> location;
+
               for (unsigned int q=0; q<quadrature_formula.size(); ++q)
                 {
-                  const Point<dim> location = fe_values.quadrature_point(q);
+                  location = fe_values.quadrature_point(q);
```

Comment:
Good point, I made this change. For the location that is written into the file I now use the midpoint of the cell face that is at the surface of the domain instead of the cell center, because I think the DT should really be outputted for a point at the surface.